### PR TITLE
update weights

### PR DESCRIPTION
**Pull request by coderobe**
similar to https://github.com/paritytech/polkadot/pull/5097 but on the release branch. weights from different benchmark run as master & release branch diverged too much to apply the exact same weights patches cleanly

### CODE REVIEW

File: runtime/polkadot/src/weights/runtime_parachains_disputes.rs [hunk 1]
```diff
@@ -16,11 +16,11 @@
 //! Autogenerated weights for `runtime_parachains::disputes`
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2022-02-25, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! DATE: 2022-03-11, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
 //! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("polkadot-dev"), DB CACHE: 1024
 
 // Executed Command:
-// target/production/polkadot
+// target/release/polkadot
```
Comment (chevdor) [inline runtime/polkadot/src/weights/runtime_parachains_disputes.rs:23]: This does not look good.

See 	[#[pallet::weight(T::SystemWeightInfo::remark(_remark.len() as u32))]](https://github.com/paritytech/polkadot/blob/975e780ae0d988dc033f400ba822d14b326ee5b9/scripts/run_benches_for_runtime.sh#L8)

**Review by chevdor — CHANGES_REQUESTED** [review]
The weights should run using the production profile.

**Comment by coderobe** [comment]
the weights should now be in sync with https://github.com/paritytech/polkadot/pull/5098

**Comment by ggwpez** [comment]
So this is now doing the same as #5098?

**Comment by coderobe** [comment]
> So this is now doing the same as #5098?

Yes

**Comment by coderobe** [comment]
Seeing that the weights have been reviewed in the pr targeting master, I'll consider them reviewed here as well. I'll be merging this once i have backported the frame_system benchapp weights from #5097 (waiting on results atm)

**Pull request merged**